wagtail/admin/views/generic/models.py

```diff
@@ -310,12 +293,9 @@ def get_columns(self):
                 else:
                     column = self._get_custom_column(field)
                 columns.append(column)
+            self.columns = columns
```

Comment:
Given that we already have `self.columns = self.get_columns()` in `setup()`, how about we remove this line and change the `try`/`except` into `if self.column is not None`/`else`?

Just thought that it's nicer to not have both the assignment and return in the same method.

We can also remove the `else` after the early return, but that will introduce some whitespace-only changes.